data/json/items/gun/8x40mm.json

```diff
@@ -74,7 +74,11 @@
     "aim_speed": 4,
     "recoil": 100,
     "durability": 9,
-    "burst": 5,
+    "modes": [
+      [ "DEFAULT", "semi-auto", 1 ],
+      [ "BURST", "2 rd.", 2 ],
```

Comment:
The scout rifle description says it only has a 2-round burst and cannot fire full auto. Can NPCs use a burst-only automatic weapon?